src/audio/volume/volume_hifi3.c

```diff
@@ -31,6 +31,7 @@ static void vol_setup_circular(const struct audio_stream *buffer)
 	AE_SETCBEGIN0(buffer->addr);
 	AE_SETCEND0(buffer->end_addr);
 }
+
```

Comment:
This patch looks like it needs squashed or it has the wrong commit message ?

Reply:
@lgirdwood maybe it is because the multiple commit and one commit.